Mathlib/Topology/Homotopy/Basic.lean

```diff
@@ -614,7 +625,7 @@ def cast {f₀ f₁ g₀ g₁ : C(X, Y)} (F : HomotopyRel f₀ f₁ S) (h₀ : f
 /-- Post-compose a homotopy relative to a set by a continuous function. -/
 @[simps!] def compContinuousMap {f₀ f₁ : C(X, Y)} (F : f₀.HomotopyRel f₁ S) (g : C(Y, Z)) :
```

Comment:
How should this be called? `HomotopyRel.map`?

Reply:
I like that name, but would like another pair of eyes. @PatrickMassot maybe?

Reply:
`compContinuousMap` suggests that we take composition with a continuous map on the right, not on the left.